Resources/Prototypes/Entities/Objects/Specific/Medical/hypospray.yml

```diff
@@ -18,6 +18,7 @@
   - type: ExaminableSolution
     solution: hypospray
   - type: Hypospray
+    onlyMobs: false
```

Comment:
Add this to the Gorlex Hypospray (and probably the Hypopen too).

Reply:
oh, I thought gorlax spray parented from hypospray. One moment

Reply:
It does, but for some reason it reuses the hypospray component